install/install.go

```diff
@@ -35,19 +37,63 @@ type Options struct {
 	Save           bool
 }
 
+type progressBar struct {
+	bar string
+	fill string
+	blank string
+	filled int
+	unfilled int
+	step int
+	done float64
+	size float64
+}
+
+var progbar progressBar
+
+func printMessage(msg string,prog bool) int {
+	if progbar.filled == 0 {
+		fmt.Printf("KARMOR-CLI INSTALL \n \n")
+	}
+	if prog {
+		progbar.filled+=progbar.step
+		progbar.unfilled-=progbar.step
+		progbar.done=(float64(progbar.filled)/progbar.size)*float64(100)
+		s:=""
+		for i:=0;i<progbar.filled;i++{
+		s=s+progbar.fill
+		}
+		for i:=0;i<progbar.unfilled;i++{
+			s=s+progbar.blank
+		}
+		progbar.bar=s
+	}
+	sp:=""
+	for i:=0;i<90;i++{
+		sp=sp+" "
+	}
+	fmt.Printf("\r"+sp+"\r"+msg+"\nKARMOR INSTALLING       ["+progbar.bar+"] : %.2f%%",progbar.done)
```

Comment:
fmt.Printf("\r"+sp+"\r"+msg+"\n Installing KubeArmor       ["+progbar.bar+"] : %.2f%%",progbar.done)

Reply:
That amount of space helps it line up with bigger messages that are printed also

Reply:
Okay, you can keep the space then, just change the message